### PR TITLE
[amd64-generic] Rebase on latest v6.1.106

### PR DESCRIPTION
**Pull request by rouming**
Just `git rebase v6.1.106`, the following commit was dropped 
```
096014789d01 ("rsi91x: Check wowlan pointer in rsi_shutdown")
```

similar fix was applied in stable branch.

### CODE REVIEW

**Comment by rucoder** [comment]
@rouming   DCO sometimes fails when rebasing. I think it should be ok to merge

**Review by rucoder — APPROVED** [review]
LGTM

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: eve-kernel build
+
+on:
+  workflow_dispatch:
+  pull_request_review:
+    types: [ submitted ]
+  push:
+    branches:
+      - "eve-kernel-amd64-v6.1.38-generic"
```
Comment (rucoder) [inline .github/workflows/publish.yml]: branch name!

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+env:
+  branch_name: "refs/heads/eve-kernel-amd64-v6.1.38-generic"
```
Comment (rucoder) [inline .github/workflows/publish.yml]: branch name!

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          make -f Makefile.eve BRANCH?=${GITHUB_REF##*/} kernel-gcc
+
+      - name: Log in to Docker Hub
+        if: ${{ github.ref == env.branch_name }}
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.RELEASE_DOCKERHUB_ACCOUNT }}
+          password: ${{ secrets.RELEASE_DOCKERHUB_TOKEN }}
+      
+      - name: Push eve-kernel-amd64-v6.1.38-generic if PR approved or pushed
```
Comment (rucoder) [inline .github/workflows/publish.yml]: branch name!

**Review by rucoder — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (rouming) [reply .github/workflows/publish.yml]: Done

File: .github/workflows/publish.yml (hunk 3 above)
Comment (rouming) [reply .github/workflows/publish.yml]: Done

File: .github/workflows/publish.yml (hunk 2 above)
Comment (rouming) [reply .github/workflows/publish.yml]: Done

**Review by rucoder — APPROVED** [review]
LGTM

**Pull request merged**